test/quantization/test_quantize_script.py

```diff
@@ -1233,80 +1232,42 @@ def _test_op_impl(self, module, data, quantized_op, tracing=False, debug=False,
+            def __init__(self, dim):
+                super(Conv, self).__init__()
+                self.conv = conv_module[dim](3, 3, 3).float()
 
             def forward(self, x):
                 return self.conv(x)
 
-        data = [(torch.rand((1, 3, 10, 10), dtype=torch.float), torch.randint(0, 1, (1,), dtype=torch.long)) for _ in range(2)]
-        model = self._test_op_impl(M(), data, "quantized::conv2d")
-        # make sure there is only one quantize_per_tensor for input
-        # and conv2d_prepack is folded
-        FileCheck().check_count("aten::quantize_per_tensor", 1, exactly=True) \
-                   .run(model.graph)
-
-        FileCheck().check_not("quantized::conv2d_prepack") \
-                   .run(model.graph)
-
-
-    @unittest.skipUnless(
-        'fbgemm' in torch.backends.quantized.supported_engines,
-        " Quantized operations require FBGEMM. FBGEMM is only optimized for CPUs"
-        " with instruction set support avx2 or newer.",
-    )
-    def test_quantized_conv3d(self):
-        class M(torch.nn.Module):
-            def __init__(self):
-                super(M, self).__init__()
-                self.conv = torch.nn.Conv3d(3, 3, 3).float()
-
-            def forward(self, x):
-                return self.conv(x)
+        options = itertools.product([1, 2, 3], [True, False])
+        input_data = {1 : self.img_data_1d,
+                      2 : self.img_data,
+                      3 : self.img_data_3d}
+        for dim, tracing in options:
+            model = self._test_op_impl(
+                Conv(dim), input_data[dim], "quantized::conv{}d".format(dim), tracing)
+            # make sure there is only one quantize_per_tensor for input
+            # and conv2d_prepack is folded
+            FileCheck().check_count("aten::quantize_per_tensor", 1, exactly=True) \
+                       .run(model.graph)
 
-        data = [(torch.rand((1, 3, 10, 10, 10), dtype=torch.float), torch.randint(0, 1, (1,), dtype=torch.long)) for _ in range(2)]
-        model = self._test_op_impl(M(), data, "quantized::conv3d")
-        # make sure there is only one quantize_per_tensor for input
-        # and conv3d_prepack is folded
-        FileCheck().check_count("aten::quantize_per_tensor", 1, exactly=True) \
-                   .run(model.graph)
+            FileCheck().check_not("quantized::conv{}d_prepack".format(dim)) \
+                       .run(model.graph)
 
-    @override_qengines
-    def test_quantized_convnd_relu(self):
+    @skipIfNoFBGEMM
+    def test_quantized_conv_relu(self):
         """tests for conv1d_relu/conv2d_relu/conv3d_relu"""
         conv_module = {1 : torch.nn.Conv1d, 2 : torch.nn.Conv2d, 3 : torch.nn.Conv3d}
 
         class ConvNdRelu(torch.nn.Module):
             def __init__(self, dim, inplace):
                 super(ConvNdRelu, self).__init__()
-                self.conv = conv_module[dim](1, 4, 2, 3).float()
+                self.conv = conv_module[dim](3, 3, 3).float()
```

Comment:
Why is the dimension changed here?

Reply:
you mean the argument? they are changed because I changed the input dimensions